### DISCUSSION TITLE
What Sample Rate is used for the command "edge-tts --write-media"

### DISCUSSION THREAD

**Discussion by rpatapa** [rany2/edge-tts#252] [Q&A] [answered]
Hi,

When we write edge-tts output into a file using --write-media <filename> parameter, what is the sample rate used to write the file? 

Is there a way to change the sample rate to any other desired value? If yes, what is the supported range.

**Comment by rany2** [comment, marked as answer]
https://github.com/rany2/edge-tts/blob/dfd4cab849a988d9587684cf3f9f9536c92b8f4d/src/edge_tts/communicate.py#L339

It's hardcoded because the API doesn't support anything better than this.

**Reply by rpatapa** [reply]
Does the issue of not supporting anything better that the cited format also implies that, we cannot generate RIFF supported audio file format?

The reason I am asking is that, it is getting difficult to play the file generated in some playback method.. reason being the file is a non-RIFF format file'

**Reply by rany2** [reply]
We cannot support RIFF. You'll need to convert it to mp3 chunk to RIFF.

**Reply by rpatapa** [reply]
Understood! Thank you!
